### ISSUE TITLE
signing up through Mathshare (separate from other OAuth)

### ISSUE THREAD

**Issue by alexrcabral** [benetech/MathShare#903]
concerns of linking google app for students with disabilities (especially VI users)

nearly all users with visual impairments will only sign in if they can keep their account separate (via Bookshare Customer Support team, context of Bookshare and Dolphin)

**Comment by jscholes** [comment]
> nearly all users with visual impairments will only sign in if they can keep their account separate (via Bookshare Customer Support team, context of Bookshare and Dolphin)

I don't agree with this blanket statement.  The Google sign in flow is reasonably accessible with a screen reader, and many users won't have an issue with it.  Particularly if, say, their school already uses GSuite and they're already signed into that.

I do think there will be groups of users who are able to use Bookshare and Dolphin's reading apps, but who may not have a Google account, or otherwise understand the concept of OAuth and why Google should be linked to non-Google activities.  But it's quite a leap to go from that to "nearly all users with visual impairments".

Bringing user accounts in-house has all sorts of security and compliance implications, so I just want to flag that this needs further discussion.
